src/gtk/toplevel.cpp

```diff
@@ -34,6 +34,10 @@
 #include "wx/recguard.h"
 #include "wx/sysopt.h"
 
+#ifdef __WXGTK3__
+    #include "wx/display.h"
```

Comment:
Doesn't seem to be needed.

Reply:
Oops, thanks for noticing this, it's a leftover from the version before Mac refactoring, will remove.